package.json

```diff
@@ -124,22 +124,22 @@
     "@editorjs/code": "2.7.0",
     "@editorjs/inline-code": "1.3.1",
     "@editorjs/marker": "1.2.2",
-    "@nrwl/angular": "13.10.5",
-    "@nrwl/cli": "13.10.5",
-    "@nrwl/eslint-plugin-nx": "13.10.5",
-    "@nrwl/jest": "13.10.5",
-    "@nrwl/linter": "13.10.5",
-    "@nrwl/node": "13.10.5",
-    "@nrwl/nx-plugin": "13.10.5",
-    "@nrwl/storybook": "13.10.5",
-    "@nrwl/workspace": "13.10.5",
+    "@nrwl/angular": "^14.5.4",
```

Comment:
never use the carrot you must always pin a specific version

Reply:
can we use @nrwl/angular": "14.4.2", ?

Reply:
sure let me try to downgrade

Reply:
packages downgraded